src/Microsoft.AspNet.SignalR.Core/Configuration/IConfigurationManager.cs

```diff
@@ -52,10 +52,16 @@ public interface IConfigurationManager
 
         /// <summary>
         /// Gets or sets a <see cref="TimeSpan"/> representing tell the client to wait before restablishing a
-        /// long poll connection after data is sent from the server. 
+        /// long poll connection after data is sent from the server.
         /// The default value is 0.
         /// </summary>
         [SuppressMessage("Microsoft.Naming", "CA1720:IdentifiersShouldNotContainTypeNames", MessageId = "long", Justification = "Longpolling is a well known term")]
         TimeSpan LongPollDelay { get; set; }
+
+        /// <summary>
+        /// Gets or sets the maximum number of scaleout mappings per scaleout stream stored on the server.
+        /// The default value is 1000000
+        /// </summary>
+        int? MaxScaleoutMappingsPerStream { get; set; }
```

Comment:
Can do that as well - was following the pattern we used for MaxIncomingWebSocketMessageSize setting...